config/webpack/environment.js

```diff
@@ -1,6 +1,8 @@
 const { environment } = require('@rails/webpacker')
 const { VueLoaderPlugin } = require('vue-loader')
-const vue = require('./loaders/vue')
+const vue =  require('./loaders/vue')
+
+environment.plugins.append('VueLoaderPlugin', new VueLoaderPlugin())
```

Comment:
Is this necessary? It's weird it worked before with our other vue component.

Reply:
Yes, it's necessary because vue-loader has been upgraded and the Webpack plugin is now required. https://vue-loader.vuejs.org/migrating.html#notable-breaking-changes